analyses/2009_03_noahwalugembe__Internet-Jones-evolution.txt

```diff
@@ -0,0 +1,32 @@
+Verification / evolution of "Internet Jones" paper #26
+Buy
+Walugembe Francis Noah
+noahwalugembe@gmail.com
+
+Introduction
+
+Third-party web tracking is the practice by which entities (“trackers”) embedded in webpages re-identify users as they browse the web, collecting information about the websites that they visit. A cording to According to  Lerner, Simpson, Kohno and Roesner, (2016) web Tracking is typically done for the purposes of website analytics, targeted advertising, and other forms of personalization (e.g., social media content). In this work I am evaluating the contribution of "Internet Jones" paper #26 starting with its insight on TrackingExcavator and a longitudinal measurement study of third-party cookie-based web tracking on Wayback Machine1. I will also show how has the third-party web tracking ecosystem evolved since its beginnings according to "Internet Jones" paper.
+
+TrackingExcavator
+
+The Wayback Machine1 contains archives of full webpages, including JavaScript, stylesheets, and embedded resources, dating back to 1996. To leverage this archive, According to  Lerner, Simpson, Kohno and Roesner, (2016) designed and implemented a retrospective tracking detection and analysis platform called TrackingExcavator which allowed them to conduct a longitudinal study of third-party tracking from 1996 to present (2016). TrackingExcavator logs in-browser behaviors related to web tracking, including: third-party requests, cookies attached to requests, cookies programmatically set by JavaScript, and the use of other relevantJavaScript APIs (e.g., HTML5 LocalStorage and APIsused in browser fingerprinting, such as enumerating installed plugins). TrackingExcavator also run on both live as well as archived versions of websites.
+
+Wayback Machine
+
+According to  Lerner, Simpson, Kohno and Roesner, (2016)
+it was discovered that The Wayback Machine provides a unique and comprehensive source of historical web data. However, it was not created for the purpose of studying third-party web tracking and is thus imperfect for that use but they stated that Nevertheless, the only way to study web tracking prior to explicit measurements targeting it is to leverage materials previously archived for other purposes which is true because it is a good approach to start from some thing than reinventing from scratch. At this point I am going to mention some of the failures identified by According to  Lerner, Simpson, Kohno and Roesner, (2016)
```

Comment:
When you are quoting work it is very important to use quotation marks and citations to make it clear that you have used the original authors words. Alternatively you can re-write evidence / claims / conclusions in your own words. Here is the passage from the original paper that is too close, in my eyes, for you to claim as your own words.

To be fair, you do say `they state that Nevertheless...` but this needs to be `they state that "nevertheless...`

Additionally, your use of "they state that" may imply that earlier parts of the sentence are your own words.

Pg 7 - Sec 4

> The Wayback Machine provides a unique and comprehensive source of historical web data. However, it was not created for the purpose of studying third-party web
tracking and is thus imperfect for that use. Nevertheless, the only way to study web tracking prior to explicit measurements targeting it is to leverage materials previously
archived for other purposes.